applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_small_adjoint_primal_parameters.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
the filename should not include "adjoint" but only "primal"

Reply:
I get the reason: maybe use "...sensitivity" as a prefix and then primal/adjoint to specify the individual settings